client/src/components/History/HistoryViews/JobHistoryModal.jsx

```diff
@@ -85,7 +97,14 @@ function JobHistoryContent({ jobDetails, jobDisplay, jobName, toggle }) {
   statusDataObj[hasFailedStatus ? 'Failure Report' : 'Last Status Message'] = (
     <Expand
       detail={hasFailedStatus ? 'Last Status Message' : 'System Output'}
```

Comment:
![Screenshot 2023-02-10 at 1 50 33 PM](https://user-images.githubusercontent.com/8287580/218186107-5e67086f-eade-4df6-baef-854fb5e15f2b.png).

We are getting job_detail during the submission so we don't have udpated status etc.  Should we just hide the last status message for v2 jobs? (We could change `main` to get more up-to-date info but I personally don't think its worth it.)